rj_gameplay/rj_gameplay/role/capture_role.py

```diff
@@ -0,0 +1,28 @@
+import stp
+from rj_gameplay.skill import capture
+
+from rj_msgs.msg import RobotIntent
+
+
+class CaptureRole(stp.role.Role):
```

Comment:
can you add a docstring here explaining what the role does? (I know in this case it's very simple)